Networking/Networking/Model/ShippingLabel/Enums/ShippingLabelPaymentCardType.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+
+/// The supported card types for Shipping Label Payment Methods.
+///
+public enum ShippingLabelPaymentCardType: GeneratedFakeable {
+    case amex
+    case discover
+    case mastercard
+    case visa
+    case paypal
+}
+
+/// RawRepresentable Conformance
+extension ShippingLabelPaymentCardType: RawRepresentable {
+    /// Designated Initializer.
+    ///
+    public init(rawValue: String) {
+        switch rawValue {
+        case Keys.amex:
+            self = .amex
+        case Keys.discover:
+            self = .discover
+        case Keys.mastercard:
+            self = .mastercard
+        case Keys.visa:
+            self = .visa
+        case Keys.paypal:
+            self = .paypal
+        default:
+            assertionFailure("Unexpected value for `ShippingLabelPaymentCardType`: \(rawValue)")
+            self = .amex
```

Comment:
Is there a possibility that we can get new kind of CardType here? maybe add `.unknown` case for that? Or even `.unknown(String)`.

ProductStatus for example:
https://github.com/woocommerce/woocommerce-ios/blob/0f15625548d2ef2859cf6baf5dce97d22e4f543e/Networking/Networking/Model/Product/ProductStatus.swift#L11

Reply:
No, there shouldn't be any possibility of an unexpected card type here. This isn't a core type that could be modified by another extension; it's only the card types that our Shipping & Tax extension supports, [defined here](https://github.com/Automattic/woocommerce-services/blob/c97384bc78254bd484485f3f781917af605788c2/client/extensions/woocommerce/woocommerce-services/views/label-settings/label-payment-method.js#L18-L25). So we should be ok without a fallback case, as far I understand.